script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -198,17 +212,17 @@ A chill runs down my spine.
 [sha:739b1e33010ab3872de68204b4b916e34ceb1555]{
 -- Page 578, Offset 7123.
 -- 真っ正面から、こちらの呼吸を止めるほどの、強い視線を向けられる。
-She stares at me head-on with such a strong gaze that I stop breathing.
+She's staring at me so intensely that I almost forget how to breathe.
 }
 [sha:8d2858b4b8e55e3645cdcd518eeed6480caf5b71]{
 -- Page 579, Offset 7124.
 -- 「そう。直死の魔眼なんて童話の中だけの話と思ってたけど、いるところにはいるものね。
-"I see. I thought the %{i}Mystic Eyes of Death Perception%{/i} were the stuff of fairy tales, but I guess they do exist.
+"I see. I thought the %{i}Mystic Eyes of Death Perception%{/i} were the stuff of fairy tales, but I guess they really do exist.
 }
 [sha:658564e232b0d93fda97a9faf0d6c0265fa39ff9]{
 -- Page 579, Offset 7125.
 -- あなたみたいな、突然変異の化物が」
-Mutated monsters like you, for example."
+Monstrous mutants like yourself, that is."
```

Comment:
hmm, I'm not sure this works.  "they" in the previous sentence as it currently is, is pretty clearly referring to the meodp, not to something abstract that can be later clarified.

Maybe something like "I guess some things are stranger than fiction."

Reply:
Hmm, I feel like this is kinda an intentional fakeout though?

Reply:
it doesn't read like a fakeout to me though though, it just reads like there's a grammatical mistake.